File: auto_swap-two-integers.py
```java
public class Main {
    public static void main(String[] args) {
        // Initialize two integers, a and b
        int a = 5;
        int b = 10;

        // Output the initial values of a and b
        System.out.println("Before swapping");
        System.out.println("A: " + a);
        System.out.println("B: " + b);

        // Swap a and b using a third integer
        int tempVal = a;  // Store the value of a in tempVal
        a = b;           // Assign the value of b to a
        b = tempVal;     // Assign the value of tempVal to b

        // Output the swapped values of a and b
        System.out.println("After swapping");
        System.out.println("A: " + a);
        System.out.println("B: " + b);
    }
}
```

File: auto_swap-two-integers.java
// Initialize two integer variables
int num1 = 10;
int num2 = 20;

// Print the initial values of num1 and num2
System.out.println("Before Swap");
System.out.println("num1 = " + num1);
System.out.println("num2 = " + num2);

// Declare a temporary variable and store the value of num1 in it
int tempVariable = num1;

// Transfer the num2 value to num1
num1 = num2;

// Now transfer the tempVariable value to num2
num2 = tempVariable;

// Print the swapped values of num1 and num2
System.out.println("After Swap");
System.out.println("num1 = " + num1);
System.out.println("num2 = " + num2);